android/build.gradle

```diff
@@ -21,7 +21,7 @@ android {
     buildToolsVersion '30.0.3'
 
     defaultConfig {
-        minSdkVersion 23
+        minSdkVersion 25
```

Comment:
is there any specific reason we are changing the minSdkVersion as this will cause a lot of compatibility issues to the users

Reply:
Fixes the below error that I wasn't able to resolve otherwise.
```
java.lang.NoSuchMethodError: No static method getOrCreate(Landroid/content/Context;)Landroidx/window/layout/WindowInfoTracker; in class Landroidx/window/layout/WindowInfoTracker; or its super classes (declaration of 'androidx.window.layout.WindowInfoTracker' appears in /data/app/~~GH_Uqsa0mDNXu7n3pF4JPg==/com.zoomustest-aDAVAWyde4HK7UFNuGjQLQ==/base.apk)
                                                                                                    	at com.zipow.videobox.conference.ui.ZmConfActivity.onCreate(ZmConfActivity.java:34)
                                                                                                    	at android.app.Activity.performCreate(Activity.java:8305)
                                                                                                    	at android.app.Activity.performCreate(Activity.java:8284)
                                                                                                    	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1417)
                                                                                                    	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3626)
                                                                                                    	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3782)
                                                                                                    	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:101)
                                                                                                    	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
                                                                                                    	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
                                                                                                    	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2307)
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:106)
                                                                                                    	at android.os.Looper.loopOnce(Looper.java:201)
                                                                                                    	at android.os.Looper.loop(Looper.java:288)
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:7872)
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method)
                                                                                                    	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
```

Reply:
Great that you found a solution for this!

Reply:
I just tried and it indeed works.
I've noticed that it also works with `minSdkVersion 24` so maybe we should use that?
But yes, the problem is that with `minSdkVersion 24` or up, we are dropping support for Android 6.

Reply:
Also, done. Thank you!

Reply:
ios custom ui version which day will be live? any body work on that

Reply:
> ios custom ui version which day will be live? any body work on that

Related issue https://github.com/mieszko4/react-native-zoom-us/issues/290#issuecomment-1896217675

Reply:
> ios custom ui version which day will be live? any body work on that

PR is up #330